subsystems/intakesubsystem.py

```diff
@@ -164,9 +165,18 @@ def periodic(self) -> None:
                 NEOBrushless.ControlMode.Velocity, constants.kIntakeSpeed * -1
             )
 
-        SmartDashboard.putNumber(
-            constants.kPivotAngleKey, self.pivotEncoder.getPosition().radians()
-        )
+        if RobotBase.isSimulation():
+            SmartDashboard.putNumber(
+                constants.kPivotAngleKey,
+                self.pivotMotor.get(Talon.ControlMode.Position)
+                / constants.kPivotGearRatio
+                * constants.kRadiansPerRevolution
+                + constants.kIntakeAngleOffset.radians(),
+            )
+        else:
+            SmartDashboard.putNumber(
+                constants.kPivotAngleKey, self.pivotEncoder.getPosition().radians()
```

Comment:
the PivotAngleKey isn't used anywhere else so I'm saying it's at a *reasonable 0* with horizontal

Reply:
I mean the pivot encoder is offset so that 0 is handoff, check line 27

Reply:
oh that has to be a mistake, because the zeroing of the pivot encoder will be unique anyways since we can't gaurentee that it's magnetic zero is inline with the any reference zero for the intake